api/controllers/apps_test.go

```diff
@@ -32,7 +34,10 @@ func TestAppList(t *testing.T) {
 }
 
 func TestAppShow(t *testing.T) {
```

Comment:
This should test the happy path, so do a single DescribeAppStackCycle("convox-test-bar").

If you want to test the unbound app logic, make another test.